pkg/model/const.go

```diff
@@ -263,4 +263,7 @@ const (
 
 	// OktetoImageTagWithVolumes is the tag assigned to an image with volume mounts
 	OktetoImageTagWithVolumes = "okteto-with-volume-mounts"
+
+	// OktetoUseStaticKubetokenEnvVar is used to opt in to use static kubetoken
+	OktetoUseStaticKubetokenEnvVar = "OKTETO_USE_STATIC_KUBETOKEN"
```

Comment:
in order to not increase the model pkg, can we perhaps declare this as const under the package that is used?

Reply:
good call, done ✅